backend/adapters/config.py

```diff
@@ -1,12 +1,23 @@
 import yaml
 from pathlib import Path
+from dotenv import load_dotenv
+import os
 
 Config = {}
 
 def load():
+    load_dotenv()
     # TODO: is there a better way to do this
```

Comment:
backend and api are independent, backend can have its own config file. This will depend on how we deploy the backend

Reply:
I'll file an issue, so we can fix it later